.gitignore

```diff
@@ -1,6 +1,27 @@
-clients/
-!clients/cli/
-clients/cli/api_*.go
+clients/go/
+clients/java/
+clients/php/
+clients/python/
+
+clients/ruby/.*
```

Comment:
maybe we could put these ignore rules also in the individual client directories when they're getting quite large per client